config/locales/en.yml

```
@@ -272,6 +272,10 @@ en:
       open: "Course opened"
       closed: "Course closed"
     add_course: "Add course"
+    are_any_alevels_required_for_this_course:
+      fieldset: "Are any A levels required for this course?"
+      answer_yes: "Yes"
+      answer_no: "No"
```

```suggestion
      fieldset: Are any A levels required for this course?
      answer_yes: Yes
      answer_no: No
```
You only need the quotes if there are certain characters on the string, like `:`

---

Actually I need the double quotes on the Yes/No because YAML parses as true or false